h2o-core/src/main/java/hex/ModelMetricsBinomial.java

```diff
@@ -270,12 +270,12 @@ private ModelMetrics makeModelMetrics(Model m, Frame f, GainsLift gl) {
 
     private GainsLift calculateGainsLift(Model m, Frame preds, Vec resp, Vec weights) {
       GainsLift gl = null;
-      try {
         gl = new GainsLift(preds.lastVec(), resp, weights);
+        if (m._parms._gainslift_bins !=0 ) {
```

Comment:
There are multiple tests failing on NPE on this line (e.g. hex.tree.xgboost.XGBoostTest#WeatherBinary...) Also you can fix the condition formatting here if you want.

Reply:
Thank you. There definitely is a scenario where the model is null - some `make` methods of `ModelMetricsBinomial` allow it. Fixed by a null check.